CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
 # Changelog
 
```

Comment:
Is this automatically generated?

Reply:
It certainly is (by standard-version) as long as we stick to the commit message convention: https://www.conventionalcommits.org/en/v1.0.0/
`npm run commit` is ur friend :slightly_smiling_face: